net-p2p/classified-ads/classified-ads-0.14-r1.ebuild

```
@@ -0,0 +1,83 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit qmake-utils virtualx xdg
```

```suggestion
EAPI=8

inherit qmake-utils virtualx xdg
```

We like to keep an empty line for consistency here.